src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -697,8 +697,7 @@ private function injectAlignmentPlaceholdersForArrow(Tokens $tokens, $startAt, $
                     if ($tokens[$i + 1]->isGivenKind([T_ARRAY, CT::T_ARRAY_SQUARE_BRACE_OPEN])) {
                         $arrayStartIndex = $tokens[$i + 1]->isGivenKind(T_ARRAY)
                             ? $tokens->getNextMeaningfulToken($i + 1)
-                            : $i + 1
-                        ;
+                            : $i + 1;
```

Comment:
I don't think these kind of changes are wanted?

Reply:
Why? This is what the fixer do, I've just fix the behavior for chaining calls, see #4896